docs/app/Examples/modules/Popup/Types/PopupExampleTitled.js

```diff
@@ -18,17 +18,13 @@ const users = [
     avatar: '/assets/images/avatar/small/matt.jpg',
   },
 ]
-/* TODO: Remove <div> wrapper after all our components will be classes */
+
 const PopupExampleTitled = () => (
   <div>
     {users.map(user => (
       <Popup
         key={user.name}
-        trigger={(
-          <div>
```

Comment:
After #2219 was merged stateless component can be used as `trigger` without warnings :+1: